assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -551,6 +551,10 @@ che.infra.openshift.route.labels=NULL
 # It has to be a valid DNS name.
 che.infra.openshift.route.host.domain_suffix=NULL
 
+# Initialize OpenShift project with Che server's service account
+# if OpenShift OAuth is enabled.
+che.infra.openshift.project.init_with_server_sa=false
```

Comment:
why false? Shouldn't it be created with che-server sa by default?

Reply:
@l0rd wdyt what default value should be?

Reply:
`che-server` sa should be the default